OutSystems/outsystems-datagrid
# ODC: Changing language load mechanism

This PR is for changing the language load mechanism, so that the changes made for ODC can be set to work.
The current mechanism depends on having the an additional module being deployed simultaneously as the library, and that will simply deploy the (js) language files into the server. However, in ODC, as there's no concept of module, only of app, it would require a complete app to be made available and deployed by the developer, simply to deploy those files.

### What was happening
* When migrating to ODC, the way the multi-language is being loaded required changes.

### What was done
* Changes to the way the load of the files is done.

### Test Steps
1. Go to a page with the DataGrid and change the locale.
2. The changes should be reflected in the grid

### Checklist
